source_control/github_release.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible. If not, see <http://www.gnu.org/licenses/>.
+
+
+DOCUMENTATION = '''
+---
+module: github
+short_description: Interact with GitHub Releases
+description:
+    - Fetch metadata about Github Releases
+version_added: 2.1
+options:
+    token:
+        required: true
+        description:
+            - Github Personal Access Token for authenticating
+    user:
+        required: true
+        description:
+            - The GitHub account that owns the repository
+    repo:
+        required: true
+        description:
+            - Repository name
+    action:
+        required: true
+        description:
+            - Action to perform
+        choices: [ 'latest_release' ]
+
+author:
+    - "Adrian Moisey"
```

Comment:
You may wish to put your github here as well

`"Adrian Moisey (@adrianmoisey)"`